lib/web/apiserver.go

```diff
@@ -1139,7 +1139,10 @@ func (h *Handler) bindDefaultEndpoints() {
 	// Create bot join tokens
 	h.POST("/webapi/sites/:site/machine-id/token", h.WithClusterAuth(h.createBotJoinToken))
 	// PUT Machine ID bot by name
+	// TODO(nicholasmarais1158) DELETE IN v20.0.0 - replaced by PUT /v2/webapi/sites/:site/machine-id/bot/:name
 	h.PUT("/webapi/sites/:site/machine-id/bot/:name", h.WithClusterAuth(h.updateBot))
+	// PUT Machine ID bot by name
```

Comment:
you should also mention what's the difference between the v1 and v2 (it was a code suggestion i got myself). [example](https://github.com/gravitational/teleport/blob/8ac68fd963bd615d958065ca159e5b599c97ad61/lib/web/apiserver.go#L895)